src/index.html

```diff
@@ -4,14 +4,148 @@
+            min="12"
+            max="100">
+        </label>
+        <br>
+        <label for="birth">Full date of birth:
+          <input type="date"
+            name="birth"
+            id="birth">
+        </label>
+        <br>
+        <label for="agreement">I accept the term of the agreement
+          <input type="checkbox"
+            name="agreement"
+            id="agreement">
+        </label>
+        <br>
+      </fieldset>
+      <br>
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="mail">E-mail:
+          <input type="email"
+            placeholder="email@example.com"
+            name="mail"
+            id="mail">
+        </label>
+        <br>
+        <label for="pass">Password:
+          <input type="password"
+            name="pass"
+            id="pass"
+            minlength="6"
+            maxlength="32">
+        </label>
+      </fieldset>
+      <br>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="love_cat">Do you love cats?
+          <label for="cat_y">
+            <input type="radio"
+              name="cat_y"
+              id="cat_y"
+              value="yes"> Yes
+          </label>
+          <label for="cat_n">
+            <input type="radio"
+              name="cat_n"
+              id="cat_n"
+              value="no">  No
+          </label>
+        </div>
+        <div>What is your favorite color?
+          <input type="color"
+            id="col"
+            name="col">
+          <label for="col">
+          </label>
+        </div>
+        <label for="times">What time do you go to bed?
+          <input type="time"
+            name="times"
+            id="times"
+            required>
+        </label>
+        <br>
+        <div class="cars">What are your favorite brand of cars?
+          <select name="auto"
+            id="auto"
+            multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+        <div class="works">How do you rate our work?
+          <input type="range"
+            name="work"
+            id="work">
+          <label for="work">
+          </label>
+        </div>
+      </fieldset>
+      <br>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label for="comment">Comments:
+          <textarea
+            name="comment"
+            id="comment"
+            cols="20"
+            rows="2"
+         ></textarea>
+        </label>
+        <br>
+        <br>
+        <div class="recommend">Would you recommend us?
+          <select name="commend" id="commend">
+            <option value="yes" selected>yes</option>
+            <option value="no">no</option>
+          </select>
+        </div>
+      </fieldset>
+      <br>
+      <input type="submit"
+        class="button"
+        value="Submit"
+      ></input>
```

Comment:
```suggestion
      <input 
        type="submit"
        class="button"
        value="Submit"
      >
```
Инпут можно без закрывающего тега писать, как ты делаешь это выше